Update Installation-for-Linux.md adding PipeWire usage. (#980)
jamulussoftware/jamulussoftware.github.io: wiki/en/Software-Manual.md

Some sound card drivers do not allow the buffer delay to be changed from within the Jamulus software.
In this case the buffer delay setting is disabled and has to be changed using the sound card driver. On Windows, press the ASIO Setup button to open the driver settings panel.

> On Linux, use the JACK configuration tool to change the buffer size or, if using PipeWire's JACK server, change the PipeWire's Quantum with its own configuration tools.

Reviewer: "if using PipeWire's JACK server, change ~~the~~ PipeWire's Quantum **parameter** with its own configuration tools"

> On Linux, use the JACK configuration tool to change the buffer size or, if using PipeWire's JACK server, change PipeWire's Quantum parameter with its own configuration tools.

Reviewer 2: ```suggestion
On Linux:
- when using `jackd`, use the `QJackCtl` to change the buffer size and restart JACK;
- when using PipeWire's JACK server, change PipeWire's Quantum parameter with its own configuration tools.
```

Author: The same here. Why jackd?
Second thing, you can use another tool to change the buffer size. Why do we have to talk about QjackCtl?

Reviewer 2: At least be consistent in how things are described.

Author: Ok thanks.

Some sound card drivers do not allow the buffer delay to be changed from within the Jamulus software.
In this case the buffer delay setting is disabled and has to be changed using the sound card driver. On Windows, press the ASIO Setup button to open the driver settings panel.

On Linux:

> - when using JACK, use the `QJackCtl` to change the buffer size and restart JACK;

Reviewer 3: ```suggestion
- when using JACK, use `QJackCtl` to change the buffer size and restart JACK;
```
